src/host/hcd.h

```diff
@@ -124,6 +124,10 @@ void hcd_int_disable(uint8_t rhport);
 // Get frame number (1ms)
 uint32_t hcd_frame_number(uint8_t rhport);
 
+// Optional MCU Data Cache control if you require this please implement the functions in your hcd_platform.c file
+TU_ATTR_WEAK extern void hcd_dcache_flush(uint32_t addr, uint32_t data_size);
```

Comment:
minor: could you remove the `extern`, since this is supposed to be implemented by hcd.

Reply:
thanks, It is only minor things, I sometimes mix thing up a bit.